# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/decibels/typescript6.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Fix build with TypeScript 6: set rootDir, use bundler module resolution
and drop the deprecated baseUrl.

--- decibels-49.6.1.orig/tsconfig.json	2026-04-12 00:04:40.000000000 +0200
+++ decibels-49.6.1/tsconfig.json	2026-09-26 16:12:07.154781091 +0200
@@ -8,14 +8,14 @@
     "allowJs": true,
     "checkJs": true,
     "outDir": "_build/tsc-out",
+    "rootDir": "./src",
     "strict": true,
-    "moduleResolution": "node",
-    "baseUrl": ".",
+    "moduleResolution": "bundler",
     "paths": {
       "*": [
-        "*",
-        "types",
-        "gi-types"
+        "./*",
+        "./types",
+        "./gi-types"
       ]
     },
     "typeRoots": [
